overrides/home.html

```diff
@@ -331,20 +319,60 @@ <h2>Follow us on Twitter / X</h2>
 {% block scripts %}
 {{ super() }}
 <script>
-  document.addEventListener("DOMContentLoaded", () => {
-    const logos = document.getElementById("usedby").getElementsByTagName("div");
-    for (let i = 0; i < logos.length - 1; i++) {
-      // Technically there's a chance two elements end up with the same
-      // random order in which case the result won't be truly random (it'll be based
-      // on the original document order). This really doesn't matter.
-      logos[i].style.order = Math.floor(Math.random() * 99);
+  document.addEventListener("DOMContentLoaded", function() {
+    const adopters = [
+      { logoPath: "images/corporate-logos/Google.svg", url: "https://www.google.com" },
+      { logoPath: "images/corporate-logos/vmware.svg", url: "https://www.vmware.com" },
+      { logoPath: "images/corporate-logos/IBM.svg", url: "https://www.ibm.com" },
+      { logoPath: "images/corporate-logos/Redhat.svg", url: "https://www.redhat.com" },
+      { logoPath: "images/corporate-logos/triggermesh_logo.svg", url: "https://www.triggermesh.com" }
+      
+    ];
```

Comment:
Can you also add the adopters here?

https://github.com/knative/community/blob/main/ADOPTERS.MD

Unfortunately, we don't have their logos handy :(

Reply:
And, can you make the whole list randomized? Not statically randomized, but randomized in the runtime?

I mean, it would be nice if we see the adopters+vendors in random order every time we refresh this page.

Reply:
We should drop trigger mesh - they're no longer in business

Reply:
The latest changes are as follows:
1. Added more corporate logos by following - https://github.com/knative/community/blob/main/ADOPTERS.MD
2. Mixed the adopters and vendors under the 'Trusted by' heading.
3. Added a second row that scrolls to the right.
4. Randomized the logo display order on each refresh.
5. Dropped Trigger mesh.

Reply:
The current list is not yet fully completed, and there are some aspect ratio fixes needed for certain images...
however, the logic has been implemented, and we simply need to continue adding the remaining logos while ensuring that the aspect ratio is maintained for all the images. 
Let me know if this approach seems acceptable, and then I can proceed with the final adjustments!

Reply:
Looks great!

As you said, some images need some work (aspect ratio, whitespace padding), but the overall logic looks good to me.

Special thanks for keeping the commit history clean!

I just noticed that padding is a little bit too much on mobile screens, but that's another story. Our website is not properly responsive.